src/plugins/unified_field_list/server/routes/index.ts

```diff
@@ -11,6 +11,9 @@ import { PluginStart } from '../types';
 import { existingFieldsRoute } from './existing_fields';
 import { initFieldStatsRoute } from './field_stats';
 
+// These routes are defined only for running integration tests under
```

Comment:
QQ: Do we even need this routes, if the only clients are integration tests but they are not used in the UI?

Reply:
@kertal Once we find a replacement for those tests, we can remove routes.